examples/ex_ttf.c

```diff
@@ -45,17 +45,36 @@ static void render(void)
     ALLEGRO_COLOR red = al_map_rgba_f(1, 0, 0, 1);
     ALLEGRO_COLOR green = al_map_rgba_f(0, 0.5, 0, 1);
     ALLEGRO_COLOR blue = al_map_rgba_f(0.1, 0.2, 1, 1);
+    ALLEGRO_COLOR purple = al_map_rgba_f(0.3, 0.1, 0.2, 1);
     int x, y, w, h, as, de, xpos, ypos;
+    unsigned int index;
     int target_w, target_h;
     ALLEGRO_USTR_INFO info, sub_info;
     const ALLEGRO_USTR *u;
+    const char *tulip = "Tulip";
+    const char *dimension_text = "Tulip";
+    const char *vertical_text  = "Rose.";
 
     al_clear_to_color(white);
 
     al_hold_bitmap_drawing(true);
    
-    al_draw_textf(ex.f1, black, 50,  50, 0, "Tulip (kerning)");
-    al_draw_textf(ex.f2, black, 50, 100, 0, "Tulip (no kerning)");
+    al_draw_textf(ex.f1, black, 50,  20, 0, "Tulip (kerning)");
+    al_draw_textf(ex.f2, black, 50,  80, 0, "Tulip (no kerning)");
+    
+    x = 50 ; y = 140;    
```

Comment:
@SiegeLord: What to do about it, though? I had to shift them up for the al_get_glyph_dimensions example. Should I extract the examples from ex_font and ex_ttf and move them to ex_glyph? Or...? Any suggestion is welcome.